docs/en/how-to/loading-fixtures.rst

```diff
@@ -106,3 +106,11 @@ then pass ``true`` to the 2nd argument of execute:
 .. code-block:: php
     <?php
     $executor->execute($loader->getFixtures(), append: true);
+
+By default the ``ORMExecutor`` will wrap the purge and the load of fixtures
+in a single transaction. You can pass ``false`` to the third argument
+to wrap the purge and the load of every fixture in its own transaction:
```

Comment:
As a user, why would I want to do that?

Reply:
I updated the text with an example, I don't know if you had another example in mind but that was the issue for me.

Reply:
> I don't know if you had another example in mind

I didn't have any reason in mind, I don't need that change myself. To be clear, your issue would be solved by disabling transactions entirely, but you'd rather have transactions on a fixture per fixture basis because… it's easier to start from there I suppose? The only advantage I can see is that will have fewer things to cleanup (you won't have to cleanup anything for the fixture that failed).

Reply:
I think it's good to still have transaction per fixture in case of failure, as you said, and yes, it's mostly because it's easier to have a flag to change this instead of disabling the transactions entirely (typically in a CI context).

Reply:
So if there was a flag to disable transactions entirely, that would fix your issue as well, right? I wonder if using a boolean here is the right thing to do. I'm foreseeing issues where people have their issues fixed by disabling transactions entirely, but not by using a `per-fixture` mode.  Maybe we should have this instead:

```php
/* Convert this to enum when dropping support for PHP 7 */
interface TransactionMode
{
    public const SINGLE = 'single';
    public const PER_FIXTURE = 'per_fixture';
    public const DISABLED = 'disabled';
}
```

Reply:
That would also solve the issue yes, what I like about having one transaction per fixture is that it's often pretty close to the business logic in HTTP context, while a single transaction for everything will never happen.